kebechet_administrator.py

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env python3
+# workflow-helpers Kebechet Administrator
+# Copyright(C) 2020 Sai Sankar Gochhayat
+#
+# This program is free software: you can redistribute it and / or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Kebechet Administrator.
+
+This script run in a workflow task to take and incoming message and decide and decides which repositores
```

Comment:
`This script run in a workflow task to take an incoming message and decides which repositories Kebechet needs to be run on and store the necessary messages to be sent.`